tests/other_test.go

```diff
@@ -1,18 +0,0 @@
-// Tests in this file are NOT run in the PR pipeline. They are run in the continuous testing pipeline along with the ones in pr_test.go
-package test
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-)
-
-func TestRunBasicExample(t *testing.T) {
```

Comment:
The PR must include a complete e2e test with a simple test OCP cluster, even if not ran in the GHA PR test